### PR TITLE
Fixed 'Featured Image' block not working

### PR DESCRIPTION
**Pull request by kyleminto**
Default empty array for post-thumbnails was causing it to not display.

## Description
**array()** was present in the add_theme_support function for post-thumbnails causing all featured image blocks to break unless individually specified.

## Motivation and Context
Fixes a core feature.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring (a code change that neither fixes a bug nor adds a feature)
- [ ] Styling (changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc))
- [ ] Development (changes that affect the build system or external dependencies)
- [ ] Documentation (changes that affect existing inline documentation)

## Checklist:

- [x] I pulled my branch from `develop`.
- [x] I am submitting my pull request to `develop`.
- [x] I have resolved any conflicts merging this pull request would create.
- [x] I have checked that there aren't other open Pull Requests for the same update/change.
- [x] My code follows the code style of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] \(Optional) My change requires a change to the documentation.
- [ ] \(Optional) I have updated the documentation accordingly.
- [ ] \(Optional) My change requires a change to the translations.
- [ ] \(Optional) I have updated the translations accordingly.
- [ ] `composer phpcs` has passed locally.
- [ ] `composer php-lint` has passed locally.
- [ ] `composer phpmd` has passed locally.
- [ ] `composer phpstan` has passed locally.
- [x] I have read the **[CONTRIBUTING](https://github.com/understrap/understrap/blob/main/.github/CONTRIBUTING.md)** document.

### CODE REVIEW

**Comment by bacoords** [comment]
@IanDelMar Curious your thoughts on this as well - I've been seeing [others have issues with post thumbnails](https://wordpress.org/support/topic/no-more-post-thumbnails/) after the 1.2.0 release 

Documentation doesn't really ask [for the second parameter](https://developer.wordpress.org/reference/functions/add_theme_support/#post-thumbnails), so I'm guessing it was a code linting issue when it was first added, but the checks all pass here.

Basically we've had a few variations here: 
`add_theme_support( 'post-thumbnails' );`
`add_theme_support( 'post-thumbnails', true );`
`add_theme_support( 'post-thumbnails', array() );`

It seems like the first may be the best?

**Comment by IanDelMar** [comment]
@bacoords without the second parameter PHP throws a warning in WP 5.0-5.2. However, if fixing a warning causes so much trouble, then it's probably best to stick to the warning in old WP versions.

As far as I remember the second parameter has never been documented...

Related to #1841 and #1863.

**Comment by IanDelMar** [comment]
Rechecked it. For me featured images show up - that's why I didn't notice that something has changed - but one can't add new featured images to any post or replace the featured image of a post. This happens with the array-version only for all supported versions. Reverting the PRs linked above and going for `add_theme_support( 'post-thumbnails' );` again is the best. While the true-version does not have the adverse effects on featured images it introduces a warning in WP > 5.2 which are more versions than just WP 5.0-5.2.

**Comment by bacoords** [comment]
That matches what I'm seeing. My plan is to just merge this fix and release it as 1.2.1 for the quick bug fix.

I'll wait until tomorrow AM (PST) and if no other issues come up from users, I'll release it then.

**Review by bacoords — APPROVED** [review]
